source/_posts/2017-11-15-code-generation-with-sourcery.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Getting stack traces in Swift on Linux
```

Comment:
We need to come up with a title.. Something like "Vapor development with Sourcery" ish?